### DISCUSSION TITLE
Does this work if HA and Podman (docker fork) are on seperate devices?

### DISCUSSION THREAD

**Discussion by wertercatt** [MichelFR/MqDockerUp#237] [Q&A] [answered]
Hiya.
In my network, Home Assistant is running on a dedicated Raspberry Pi. I have Podman (a fork of Docker) running on a seperate NixOS server. Would this tool work for monitoring my containers?

**Comment by MichelFR** [comment, marked as answer]
yes

MqDockerUp checks the running container on the host and send them to a MQTT server which could be anywhere.
Homeassistant connect to the MQTT server.

I cannot assure that it works in combination with podman since i only tested on docker.
